lib/philomena/tags.ex

```diff
@@ -557,6 +626,27 @@ defmodule Philomena.Tags do
     |> Repo.all()
   end
 
+  @doc """
+  Accepts IDs of tags and increments their `images_count` by 1.
+  """
+  @spec update_images_counts(term(), integer(), [integer()]) :: integer()
+  def update_images_counts(repo, diff, tag_ids) do
+    case tag_ids do
```

Comment:
could this `case` statement not be eliminated by using pattern matching in the function definition like `def update_image_counts(repo, diff, tag_ids) when length(tag_ids) == 0, do: 0`

Reply:
i'd also rename the function to `update_image_counts`, instead of plural `images`